docker/Dockerfile.truffle

```diff
@@ -0,0 +1,19 @@
+FROM node:9.4.0-alpine
```

Comment:
Why is this file in a `docker` directory?

Reply:
Idea is to not clog up the root when we potentially add more `Dockerfile`s, but I think I'll move it back out for now